src/main/java/org/fogbowcloud/manager/core/models/orders/instances/ComputeOrderInstance.java

```diff
@@ -31,6 +32,11 @@ public ComputeOrderInstance(String id, String hostName, int vCPU, int memory, In
         this.sshExtraPorts = sshExtraPorts;
     }
 
+    // TODO: This method was created to make getInstance() more easy to implement
```

Comment:
getInstance from which class?

Reply:
getInstance is to get a Virtual Machine from Openstack. It's not related to class' instances.